### ISSUE TITLE
Issue with model prediction on a single string

### ISSUE THREAD

**Issue by AnshulArya** [aub-mind/arabert#28]
I am using [AraBERT_Text_Classification_with_HF_Trainer_Pytorch_GPU.ipynb](https://github.com/aub-mind/arabert/blob/master/examples/AraBERT_Text_Classification_with_HF_Trainer_Pytorch_GPU.ipynb) for model training and used my dataset and while predicting on a single string as in cell 83 of the mentioned Ipynb file, I modified my code 

``` 
sentence_a = 'test string' #this will contain arabic text string for classification task
inputs = tokenizer.encode_plus(test_string, return_tensors='pt', add_special_tokens=True)
token_type_ids = inputs['token_type_ids']
input_ids = inputs['input_ids']
attention = model(input_ids, token_type_ids=token_type_ids)[-1]
input_id_list = input_ids[0].tolist() # Batch index 0
tensor = torch.LongTensor(input_id_list)
tensor = tensor.unsqueeze(0)
prediction = torch.nn.Softmax(model(tensor))
```

I am not able to extract the array from the softmax function saved in the prediction variable.
Is there any other way to predict the classes from a single string or can you give any other suggestion ???

**Comment by WissamAntoun** [comment]
try this:

```python
sentence_a = 'test string' #this will contain arabic text string for classification task
inputs = tokenizer.encode_plus(sentence_a, return_tensors='pt', add_special_tokens=True)
token_type_ids = inputs['token_type_ids']
input_ids = inputs['input_ids']
logits = model(input_ids, token_type_ids=token_type_ids)[0]
print(logits)
prediction = torch.nn.Softmax(dim=1)(logits)
print(prediction)
```
output:
```
tensor([[ 0.2499, -0.1405]], grad_fn=<AddmmBackward>)
tensor([[0.5964, 0.4036]], grad_fn=<SoftmaxBackward>)
```

You can also try to pass a dataloader to the `trainer.predict()` function with random labels
